### ISSUE TITLE
[NTR] B6.SJL-Slc6a3tm1.1(cre)Bkmn/J knock-in mice

### ISSUE THREAD

**Issue by rachadele** [HumanCellAtlas/ontology#92]
UCSC would like to request a new HCA mouse ontology for the following strain:

[https://www.jax.org/strain/006660](url)

"DATIREScre knock-in mice have Cre recombinase expression directed to dopaminergic neurons, without disrupting endogenous dopamine transporter expression. These mice may be useful for studying gene function in dopaminergic neurons, such as drug addiction, Parkinson's disease and Attention Deficit-Hyperactivity Disorder (ADHD)."

We're wondering what term label/parent term would be most appropriate? Thanks!

**Comment by paolaroncaglia** [comment]
Hi @rachadele ,
Thanks. I moved this request to the EFO tracker and I tagged you there: https://github.com/EBISPOT/efo/issues/1266
I'll close this ticket now.
